doc/nrf/ug_bootloader_external_flash.rst

```diff
@@ -3,13 +3,57 @@
 Using external flash memory partitions
 ######################################
 
+
 When using MCUboot, you can store the storage partition for the secondary slot in the external flash memory, using a driver for the external flash memory that supports the following features:
 
 * Single-byte read and write.
 * Writing data from the internal flash memory to the external flash memory.
 
+.. note::
+
+   The Partition Manager will only support run-time access to flash partitions defined in regions placed on placed on external flash devices that have drivers compiled in.
+   Partition Manager can not determine which partitions will be used at runtime, it can only determine which have or have not drivers enabled, so it will decide to only include into, partition map, partitions on flash devices with drivers enabled.
+   Lack of partition access will cause MCUboot to fail at run-time.
+   For more details on configuring and enabling access to external flash devices please refer to :ref:`pm_external_flash`.
+
 The QSPI NOR flash memory driver supports these features, and it can access the QSPI external flash memory of the nRF52840 DK and nRF5340 DK.
 
 See the test in :file:`tests/modules/mcuboot/external_flash` for reference.
 This test passes both devicetree overlay files and Kconfig fragments to the MCUboot child image through its :file:`child_image` folder.
 See also :ref:`ug_multi_image_variables` for more details on how to pass configuration files to a child image.
+
+Troubleshooting
+***************
+
+MCUboot could fail, reporting the following error:
+
+.. code-block:: console
+
+   *** Booting Zephyr OS build v3.1.99-ncs1-... ***
+   I: Starting bootloader
+   W: Failed reading sectors; BOOT_MAX_IMG_SECTORS=512 - too small?
+   W: Cannot upgrade: not a compatible amount of sectors
+   I: Bootloader chainload address offset: 0x10000
+   I: Jumping to the first image slot
+
+This error could depend on the following causes:
```

Comment:
```suggestion
This error could be caused by one the following causes:
```

Reply:
Will shorten the sentence so that "cause" would not appear twice.

Reply:
Replaced with
```
A cause of this error could be:
```